src/MongoDB.Driver.Core/Core/Bindings/CoreSession.cs

```diff
@@ -127,10 +127,6 @@ public void AbortTransaction(CancellationToken cancellationToken = default(Cance
                     ExecuteEndTransactionOnPrimary(firstAttempt, cancellationToken);
                     return;
                 }
-                catch (Exception exception) when (ShouldIgnoreAbortTransactionException(exception))
-                {
-                    return; // ignore exception and return
-                }
```

Comment:
Not sure why this was here. But any exceptions that are not retried will still be ignored (see line 136 on right).

Reply:
Ack